Library/Homebrew/software_spec.rb

```diff
@@ -306,9 +306,7 @@ def initialize(formula, spec)
 
     filename = Filename.create(formula, tag, spec.rebuild).bintray
 
-    # TODO: this will need adjusted when if we use GitHub Packages by default
```

Comment:
Please re-add this comment.

Reply:
How does it need to be adjusted when use GitHub Contain Registry by default in Homebrew/core?

Reply:
All the Bintray logic will need to be removed and this will need to be the default path.

Reply:
```ruby
    path, resolved_basename = if spec.root_url.match?(GitHubPackages::URL_REGEX)
      ["#{@name}/blobs/sha256:#{checksum}", filename]
    else
      filename
    end
```
Logic specific to Bintray will need to be removed, but this particular code is not specific to Bintray. It will need to remain as is to support third-party tap that use a `root_url` to store files on a normal non-GHCR HTTPS server.